docs/release-info/release-notes.md

```diff
@@ -57,46 +57,460 @@ For tips about how to write a good release note, see [Release notes](https://git
+- Improved window function offsets for `ArrayListRowsAndColumns` [#16718](https://github.com/apache/druid/pull/16718)
+- Improved the fallback strategy when the Broker is unable to materialize the subquery's results as frames for estimating the bytes [#16679](https://github.com/apache/druid/pull/16679)
+- Improved how Druid executes queries that contain a LIMIT clause [#16643](https://github.com/apache/druid/pull/16643)
+- Improved the code style of `NestedDataOperatorConversions` to be consistent for each `SqlOperatorConversion` [#16695](https://github.com/apache/druid/pull/16695)
+- Improved window functions so that they reject multi-value dimensions during processing instead of failing to process them [#17002](https://github.com/apache/druid/pull/17002)
+- Improved async query by increasing its timeout to 5 seconds [#16656](https://github.com/apache/druid/pull/16656)
+- Improved error message when the requested number of rows in a window exceeds the maximum [#16906](https://github.com/apache/druid/pull/16906)
+- Improved numeric aggregations so that Druid now coerces complex types to number when possible, such as for `SpectatorHistogram` [#16564](https://github.com/apache/druid/pull/16564)
+- Improved query filtering to correctly process cases where both an IN expression and an equality (`=`) filter are applied to the same string value [#16597](https://github.com/apache/druid/pull/16597)
+- Improved the speed of SQL IN queries that use the SCALAR_IN_ARRAY function [#16388](https://github.com/apache/druid/pull/16388)
+- Updated the deserialization of dimensions in GROUP BY queries to operate on all dimensions at once rather than deserializing individual dimensions [#16740](https://github.com/apache/druid/pull/16740)
+- Fixed an issue that caused `maxSubqueryBytes` to fail when segments had missing columns [#16619](https://github.com/apache/druid/pull/16619)
+- Fixed an issue with the array type selector that caused the array aggregation over window frame to fail [#16653](https://github.com/apache/druid/pull/16653)
+- Fixed support for native window queries without a group by clause [#16753](https://github.com/apache/druid/pull/16753)
+
 ### Cluster management
 
+#### Added cluster configuration for multivalue handling
+
+Added  an optional cluster configuration property `druid.indexing.formats.stringMultiValueHandlingMode` which overrides `SORTED_SET` for string dimensions. Possible settings are `SORTED_SET` (default), `SORTED_ARRAY`, `ARRAY`.
+
+[#16822](https://github.com/apache/druid/pull/16822)
+
+#### API for cluster-level compaction configuration
+
+Druid now offers an API to update the cluster-level compaction dynamic configuration:
+
+```
+POST /druid/coordinator/v1/config/compaction/cluster
+```
+
+Example payload:
+
+```json
+{
+     "compactionTaskSlotRatio": 0.5,
+     "maxCompactionTaskSlots": 10,
+     "engine": "msq"
+}
+```
+This API deprecates the older API `/druid/coordinator/v1/config/compaction/taskslots`.
```

Comment:
Add this to a separate "Deprecated" section to make it easier to identify all the deprecated items?

Reply:
I think it is already included in the list of `Deprecations` at the top.